src/Azure/AzureAD/Authentication.AzureAD.UI/test/AzureADAuthenticationBuilderExtensionsTests.cs

```diff
@@ -510,4 +486,12 @@ public void AddAzureAD_SkipsOptionsValidationForNonAzureOpenIdConnect()
 
         Assert.NotNull(openIdConnectOptions.Get("other"));
     }
+
+    private IServiceCollection GenerateServicesForTest()
```

Comment:
really nitty (don't feel obligated to do this): GenerateTestServices? But I'm not a huge fan of method names as sentences without spaces.